MultiServer.py

```diff
@@ -746,7 +746,8 @@ async def on_client_disconnected(ctx: Context, client: Client):
 
 
 async def on_client_joined(ctx: Context, client: Client):
-    update_client_status(ctx, client, ClientStatus.CLIENT_CONNECTED)
+    if ctx.client_game_state[client.team, client.slot] not in {ClientStatus.CLIENT_READY, ClientStatus.CLIENT_PLAYING}:
```

Comment:
I'd prefer if this operated on numerical values instead, such that if the status is not higher, raise it.

Reply:
the background is that it's possible more states are added later and then it's easy to miss updating lists like the above.

Reply:
I opted to check that the status is only "not connected" so if newer status get added it doesn't matter, and still not using some arbitrary value.